.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
+  - "6"
```

Comment:
Node 4.x is still supported by ember-cli, so shouldn't we aim to maintain support for it?